test/CQRS/LeanCode.CQRS.Security.Tests/ClaimsPrinicipalTests.cs

```diff
@@ -12,6 +12,11 @@ namespace LeanCode.CQRS.Default.Tests.Security;
 
 public class ClaimsPrincipalTests
```

Comment:
Why the tests ended up here?

Reply:
It kinda was related to `ClaimsPrincipal` as we extract user id from it, anyway moved it to separate project.